src/main.ts

```diff
@@ -254,190 +105,33 @@ export const processIntent = async (intent: Intent) => {
 
   console.log(`${ts()} Bundle ${bundleLabel}: Generating Intent`);
 
-  const BEARER_TOKEN = process.env.ORCHESTRATOR_BEARER_TOKEN;
-
-  // const { data: orderCost } = await axios.post(
-  //   `${process.env.ORCHESTRATOR_API_URL}/intents/cost`,
-  //   {
-  //     ...convertBigIntFields({
-  //       ...metaIntent,
-  //       tokenTransfers: metaIntent.tokenTransfers.map(
-  //         (transfer: TokenTransfer) => ({
-  //           tokenAddress: transfer.tokenAddress,
-  //         }),
-  //       ),
-  //     }),
-  //   },
-  //   {
-  //     headers: {
-  //       "x-api-key": process.env.ORCHESTRATOR_API_KEY!,
-  //       Authorization: `Bearer ${BEARER_TOKEN}`,
-  //     },
-  //   },
-  // );
-
-  // console.dir(orderCost, { depth: null });
-
-  // const orderPath = await orchestrator.getOrderPath(
-  //   metaIntent,
-  //   targetSmartAccount.account.address,
-  // );
-  //
+  const transaction = await rhinestoneAccount.sendTransaction({
```

Comment:
it could be nicer to use prepare, sign and submit so we can log how much time each is taking

Reply:
We don't export the `prepareTransactionAsIntent` https://github.com/rhinestonewtf/sdk/blob/main/src/execution/index.ts#L284